### DISCUSSION TITLE
CAPS LOCK BEHAVIOUR DISABLED IN GNOME - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#37] [General]
# CAPS LOCK BEHAVIOUR DISABLED IN GNOME - Alan Pope's blog

SOMETIMES I PRESS THE CAPS LOCK KEY BY ACCIDENT, USUALLY WHILE PLAYING A VIDEO GAME. I THEN FIND MYSELF UNABLE TO TYPE A PASSWORD OR I AM ACCUSED OF SHOUTING ONLINE.
OVER COFFEE, MY FRIEND MARTIN EXPLAINED THAT IT’S POSSIBLE TO DISABLE THE CAPS LOCK KEY COMPLETELY IN LINUX. I’D NEVER CONSIDERED DOING THIS, MISTAKENLY THINKING THE ONLY OPTIONS WERE TO REMAP IT TO SOMETHING ELSE.
IT TURNS OUT GNOME TWEAKS HAS AN OPTION.

https://popey.com/blog/2023/09/caps-lock-behaviour-disabled-in-gnome/

**Comment by Tinche** [comment]
I keep my caps lock remapped to Esc, helps with Vim is just generally useful.

**Comment by skellat** [comment]
ALAN, YOU ARE GIVING ME FLASHBACKS TO WHEN I WORKED ON AN ANCIENT IBM MAINFRAME THAT DID NOT HAVE A CONCEPT OF LOWER CASE LETTERS

**Comment by neffscape** [comment]
LOL

**Comment by a-b** [comment]
Remapping the Caps Lock key to function as Control can indeed be a game changer for Vim users as well as others who spend a lot of time in a shell environment. The remapping can make keyboard shortcuts more ergonomic and faster to access, reducing the distance your fingers need to travel. In Vim, the Esc key is used frequently to switch between different modes, but its location on the keyboard can be somewhat inconvenient. The remappings `ctrl + [` and `ctrl + c` provide alternatives to pressing `Esc` that may be easier to reach, especially when Control is mapped to a more convenient key like Caps Lock. Over time, these small optimizations can add up, leading to a smoother and potentially faster workflow.

**Comment by rhys-the-davies** [comment]
This is a very silly post. I like it.
